External/FEXCore/Source/Interface/Core/JIT/x86_64/JITClass.h

```diff
@@ -51,8 +51,8 @@ namespace FEXCore::CPU {
 using namespace Xbyak::util;
 const std::array<Xbyak::Reg, 9> RA64 = { rsi, r8, r9, r10, r11, rbp, r12, r13, r15 };
 const std::array<std::pair<Xbyak::Reg, Xbyak::Reg>, 4> RA64Pair = {{ {rsi, r8}, {r9, r10}, {r11, rbp}, {r12, r13} }};
-const std::array<Xbyak::Reg, 11> RAXMM = { xmm0, xmm1, xmm2, xmm3, xmm4, xmm5, xmm6, xmm7, xmm8, xmm9, xmm10 };
```

Comment:
Why is xmm0 removed here? We have something like xmm11-xmm15 reserved for temps?

Reply:
xmm0 is used as the return value in functions, I've added xmm11 to the list and changed the code to use xmm0 as temp.